lib/ProductOpener/APITest.pm

```diff
@@ -125,6 +126,17 @@ sub create_user ($ua, $args_ref) {
 	return;
 }
 
+sub create_product ($ua, $product_fields) {
+	my %fields;
+	while (my ($key, $value) = each %{$product_fields}) {
```

Comment:
Alex suggested a better way to concatenate one hash into another, something like %a = (%b, %c);